synth.py

```diff
@@ -33,10 +33,11 @@
             'kms', version,
             generator_args={
                 "grpc-service-config": f"google/cloud/kms/{version}/cloudkms_grpc_service_config.json",
-                "package-name": f"@google-cloud/kms"
+                "package-name": f"@google-cloud/kms",
```

Comment:
nit: no need to have `f"  "` here (it's similar to JavaScript backtick string), since you don't have any variables there.

Reply:
cool! removed it! Thank you.